src/main/java/com/cloudogu/scm/commitmessagechecker/config/RepositoryConfigHalEnricher.java

```diff
@@ -65,7 +65,10 @@ public void enrich(HalEnricherContext context, HalAppender appender) {
 
   private boolean shouldAppendLink(Repository repository) {
     return !configStore.getGlobalConfiguration().isDisableRepositoryConfiguration()
-      && RepositoryPermissions.custom(Constants.NAME, repository).isPermitted();
+      && (
+        RepositoryPermissions.custom(Constants.READ_COMMIT_MESSAGE_CHECKER_PERMISSION, repository).isPermitted() ||
+          RepositoryPermissions.custom(Constants.WRITE_COMMIT_MESSAGE_CHECKER_PERMISSION, repository).isPermitted()
```

Comment:
I would wrap this check into new method like `RepositoryPermissions.mayRead(repository)`. You can use this in the service and it should increase the readability.